### ISSUE TITLE
Folia support

### ISSUE THREAD

**Issue by h14turkiye** [Hazebyte/CrateReloaded#592]
Although Folia is still experimental, with the amount of successful tests and improvement it is showing, it is a good candidate for porting in my opinion.

**Comment by LudovicAns** [comment]
Yes, porting to Folia is going to become inevitable in the coming months/years.
Please, plan for a porting.
